main.go

```diff
@@ -110,10 +107,10 @@ Example:
 	app.Flag("aws-nat-cidr-block", "AWS Provider requires to specify NAT-CIDR-Blocks for each AZ to have a NAT gateway in. Each should be a small network having only the NAT GW").StringsVar(&cfg.NatCidrBlocks)
 	app.Flag("aws-az", "AWS Provider requires to specify all AZs to have a NAT gateway in.").StringsVar(&cfg.AvailabilityZones)
 	app.Flag("stack-termination-protection", "Enables AWS clouformation stack termination protection for the stacks managed by the controller.").BoolVar(&cfg.StackTerminationProtection)
-	app.Flag("flush-interval", "Minimum interval to call provider on change events.").Default("5s").DurationVar(&flushToProviderInterval)
```

Comment:
maybe better to have the flush interval, but show a warning if set and default to 0

Reply:
This is IMO handled by the resync interval.

Reply:
I meant only to log a warning not that you use the value. Basically it’s the public interface that you break if you delete the flag, if you ignore the flag it’s a non breaking change.